csharp/ql/src/experimental/Security Features/campaign/Solorigate/SwallowEverythingExceptionHandler.qhelp

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>The malicious code was wrapped in generic exception catch blocks that lacked any statements</p>
```

Comment:
```suggestion
    <p>In Solorigate, the malicious code was wrapped in generic exception catch blocks that lacked any statements. This made it harder to find any suspicious runtime exceptions.</p>
```

Added some more context, again from the "Solorigate-readme" document.

Reply:
BTW. JBW helped to rewrite many of the descriptions & code comments  based on the suggestions here. I checked in all the fixes. Please let us know if further changes are necessary. Thanks.